js/lib/baseDomain.js

```diff
@@ -2,7 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-const punycode = require('punycode')
+const punycode = require('punycode/')
```

Comment:
Is this intentional? Why the `/` after?

Reply:
yes it is, because if we define it like we did before it assumes that we want to get node `punnycode` which is deprecated https://nodejs.org/api/punycode.html. This way you specify that you are referring to npm package.